src/ontology/mondo-ingest.Makefile

```diff
@@ -172,6 +172,7 @@ update-jinja-sparql-queries: python-install-dependencies
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+<<<<<<< HEAD
```

Comment:
seems  like a git merge gone wrong

Reply:
Thanks; I think that's the first time i've ever messed one of those up. fixed